charters/wot-wg-charter-draft-2019.html

```diff
@@ -178,39 +186,369 @@ <h2>Scope</h2>
         <p><i class="todo">Brief background of landscape, technology, and relationship to the Web, users, 
         developers, implementers, and industry.</i></p>
         -->
-        <p>The Working Group has X items in scope:</p>
+        <p>The Working Group has the following items in scope:</p>
         
         <div class="summary">
           <p style="text-align:left;font-style:italic">Scope Summary</p>
           
           <dl>
-            <dt><strong>Item 1:</strong></dt>
-            <dd>summary of item 1.</dd>
-            <dt><strong>Item 2:</strong></dt>
-            <dd>summary of item 2.</dd>
+            <dt><strong>TD Security Schemes:</strong></dt>
```

Comment:
Suggest to change the order, i.e. start with 
- use cases, requirements, architecture 
- link relation types
- profiles
- templates
- complex interactions
- discovery
- identity management
- vocabulary extensions
- ...

Reply:
Reordering is fine.   Even better would be grouping into a smaller number of more generic categories...

Reply:
done.